pkg/virtctl/console/console.go

```diff
@@ -93,19 +93,53 @@ func (c *Console) Run(cmd *cobra.Command, args []string) error {
 	writeStop := make(chan error)
 	readStop := make(chan error)
 
+	// Wait until the virtual machine is in running phase, user interrupt or timeout
+	runningChan := make(chan error)
+	timeoutChan := time.Tick(time.Duration(timeout) * time.Minute)
```

Comment:
this ticker goroutine never stops though we don't need it running while interacting with the console. I think we should switch to time.NewTicker and explicitly Stop() it when it's no longer needed.

Reply:
Done